frontend/src/Student/pages/Scholarships.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Container, Row, Col } from 'reactstrap'
+
+import ViewScholarshipsTab from '../containers/ViewScholarshipsTab'
+
+const addIdPropFromQueryParams = () => {
+  return window.location.search.split('=')[1]
+}
+
+const Scholarships = () => (
+  <Container>
+    <Row>
+      <Col>
+        <ViewScholarshipsTab id={addIdPropFromQueryParams()}/>
```

Comment:
takes the portfolio id from the URL parameters and passes it as a property to the ```ViewScholarshipsTab``` container, which in turn passes the appropriate parameters to its underlying components.